README.md

```diff
@@ -1,2 +1,21 @@
 # open-source-funders
-Name and descrition TBA
+
+> Name and description TBA
+
+## Background
+
+Funding in open source is a continual problem. For a project to survive beyond the momentary interest of its maintainer, it needs funds. Today, there are more ways of funding open source projects and contributors than ever: GitHub Sponsors, Open Collective, npm.fund, Patreon, and so on. Thousands of projects have made it clear they need funds; hundreds of sponsors have helped out. But it's not always clear what companies do sponsor Open Source, and which companies don't. It's also not obvious which companies sponsor who on what platform. And while tens of thousands of companies have GitHub organization profiles, we suspect that only a few hundred companies actively sponsor the code that they use.
+
+This project highlights companies that have contributed to the open source ecosystem through funds directly. We showcase companies that have given money to open source projects through any open source platform, and we monitor the different ways they do so: either through direct payouts, sponsorships, paying for talented engineers to work full-time on open source, funding meetups or conferences, and so on.
+
+Ambitious? Yes! Hopefully this project will enable open source maintainers to do more with their efforts, though, by showing which companies are likely to be interested in donating money, and by showing what platforms they use.
```

Comment:
I’d drop `Ambitious? Yes! `

Reply:
What is the goal of this paragraph (and more generally, this project)? If the goal is to enable OSS maintainers to find potential sponsors, say that more clearly.
I propose to instead say that we hope to make the companies that do sustain, more visible. In order to celebrate them.
That this can be used for OSS maintainers to find sponsors (through the discussed tooling), can be inferred

Reply:
Fair. I've changed it. 

> What is the goal of this paragraph (and more generally, this project

That's what I'm trying to suss out! :D There are really multiple goals: to showcase companies, to encourage other companies to contribute, and to make it easier for devs to see who does contribute.

Reply:
That’s a good list!